pkgs/development/python-modules/aiojobs/default.nix

```diff
@@ -0,0 +1,43 @@
+{ buildPythonPackage
+, fetchPypi
+, aiohttp
+, pytest
+, pytest-aiohttp
+, pygments
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "aiojobs";
+  version = "0.2.2";
+  format = "flit";
```

Comment:
Hm, setting format to flit prevents the python2.7 build.

```
❯ nix-build -A python2Packages.aiojobs
error: flit-2.3.0 not supported for interpreter python2.7
```

 I wonder if there should be an explicit `disabled = !isPy3k` on the package.

Reply:
I'm of the opinion that we shouldn't bend over backwards for an EOL interpreter. marking it disabled is fine

Reply:
Can you make a code suggestion as to how to disable it.

Reply:
```suggestion
  format = "flit";
  disabled = isPy27;
```